CHANGELOG.md

```diff
@@ -1,5 +1,122 @@
 # Versions
 
+## v1.0.0-beta.3
+
+_Sep 3, 2025_
+
+### General changes
+
+- **Breaking change:** Base UI event details.
+  Custom event callbacks provide BaseUIEventDetails object as their second parameter.
+  This object contains the source event, reason and methods to customize the behavior (where applicable).
+  (#2382) by @atomiks
+
+### Alert Dialog
+
+- **Breaking change:** Support `initialFocus` and `finalFocus` functions.
+  The `initialFocus` and `finalFocus` props can be functions that return DOM elements to focus.
+  (#2536) by @atomiks
```

Comment:
Should mention that the element must be returned directly (`() => ref.current`), rather than returning the ref itself